src/modules/fw_att_control/fw_att_control_params.c

```diff
@@ -359,7 +359,7 @@ PARAM_DEFINE_FLOAT(FW_W_RMAX, 30.0f);
  * Roll rate feed forward
  *
  * Direct feed forward from rate setpoint to control surface output. Use this
- * to obtain a tigher response of the controller without introducing
+ * to obtain a higher response of the controller without introducing
```

Comment:
this was probably intended to be "tighter response" - does higher mean the same as tighter in this context?

Reply:
I suppose it would be more correct and easy for understand.
In addition, Google Translator marks this expression as an error, which can mislead the user.

Reply:
Excellent. Please change to "tighter" then.

Reply:
@hamishwillee Could you tell me how to change the proposed changes? because I never did it before)